tests/common/test_cgroupstelemetry.py

```diff
@@ -438,47 +469,59 @@ def test_cgroup_is_tracked(self):
         self.assertFalse(CGroupsTelemetry.is_tracked("not_present_dummy_extension_{0}".format(i), 'cpu'))
         self.assertFalse(CGroupsTelemetry.is_tracked("not_present_dummy_extension_{0}".format(i), 'memory'))
 
+    @skip_if_predicate_false(i_am_root, "This test will only run as root")
     @skip_if_predicate_false(CGroupConfigurator.is_cgroups_supported, "CGroups not supported in this environment")
     def test_telemetry_with_tracked_cgroup(self):
-        num_polls = 5
-        name = "test-cgroup"
+        CGroupsTelemetry.cleanup()
```

Comment:
since CGroupsTelemetry is a singleton shared by all these tests, should you do the cleanup in the test setUp method